test/evp_libctx_test.c

```diff
@@ -363,8 +371,9 @@ static int test_cipher_reinit(int test_id)
         || !TEST_int_eq(EVP_EncryptUpdate(ctx, out2, &out2_len, in, sizeof(in)),
                         ccm ? 0 : 1)
         || !TEST_true(EVP_EncryptInit_ex(ctx, NULL, NULL, NULL, iv))
-        || !TEST_int_eq(EVP_EncryptUpdate(ctx, out3, &out3_len, in, sizeof(in)),
-                        ccm || siv ? 0 : 1))
+        || (!no_null_key &&
```

Comment:
??? The comment say skipping the init call with NULL key but this is skipping the update ???

Reply:
Fixed